var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -217,6 +220,12 @@ def configure_args(self):
         else:
             args.append('--enable-fast=all')
 
+        if '+cuda' in self.spec:
+            args.extend([
+                '--enable-cuda',
+                '--with-cuda={0}'.format(spec['cuda'].prefix)
+            ])
```

Comment:
Are you sure? `--disable-cuda` works just fine. All options that have an `--enable` flag also have a `--disable` or `--enable-{0}=no` flag. Also, all options that have `--with` have a corresponding `--without`. At least, that's my understanding of Autotools.